lib/linguist/languages.yml

```diff
@@ -8245,6 +8259,7 @@ YAML:
   - ".mir"
   - ".reek"
   - ".rviz"
+  - ".sim"
```

Comment:
We need a sample for this language if you're going to add this extension here.

Reply:
> We need a sample for this language if you're going to add this extension here.

I'm not 100 % what you meant - was this about the Simula language, or about the YAML fallback?

I made 2 updates to the pull request:

1) Pushed 5 more Simula language code samples
2) Added an example of an [existing `.sim` file on GitHub](https://github.com/ra4king/CircuitSim/blob/6ba0ee8e7631d67dbedf015c27b1bc2dc0ccc78c/examples/Xor.sim) which would make sense to code-highlight as YAML (it's JSON, but all JSON is valid YAML).

It would be nice if linguist supported a "noop fallback" configuration, where - if we can't positively identify a `.sim` file as valid Simula code - it would just be treated as unknown text format.

Reply:
I was referring to the YAML entry. You added it but didn't add a sample.

I see you've now added samples but you've now also added samples that are too big. Please remove any that are suppressed in the diff and any that aren't real world uses.

Reply:
Ok, I removed the samples!